app/component/itinerary/navigator/NaviUtils.js

```diff
@@ -720,5 +720,5 @@ export const LEGTYPE = {
 };
 
 export const withRealTime = (rt, children) => (
-  <span className={cx('bold', { realtime: rt })}>{children}</span>
+  <span className={cx({ bold: rt }, { realtime: rt })}>{children}</span>
```

Comment:
Could be simplified into 
```suggestion
  <span className={cx({ bold: rt, realtime: rt })}>{children}</span>
```